node/src/Commands.ts

```diff
@@ -21,6 +21,25 @@ function isLargeCommand(args: string[]) {
     return false;
 }
 
+/**
+ * Represents a condition to the `SET`, `ZADD` and `JSON.SET` commands.
+ */
+export enum ConditionalChange {
+    /**
+     * Specifies the conditional change behavior:
+     * - "onlyIfExists": Set the value only if the key o path already exists. Equivalent to `XX` in the Redis API.
```

Comment:
why do you repeat the documentation of the enum options here? is it nodejs enum doc convention?

Reply:
you're right, my bad